app/controllers/users_controller.rb

```diff
@@ -1,21 +1,3 @@
 class UsersController < ApplicationController
-  def show
```

Comment:
Whats the reasoning behind deleting the User controller? Does it still exist somewhere else? Do we need it?

Reply:
We didn't need it at this point but it's been added back in on the other pull request (because we did need it again)